### ISSUE TITLE
[question] how to change opacity

### ISSUE THREAD

**Issue by zhangpy** [google/gxui#100]
I want to set opacity of label text. Is it possible?

**Comment by ben-clayton** [comment]
I looked into this, and indeed there was a bug. This is fixed with 22f19f4e7.

Sample of changing text color and alpha:

```go

package main

import (
	"time"

	"github.com/google/gxui"
	"github.com/google/gxui/drivers/gl"
	"github.com/google/gxui/gxfont"
	"github.com/google/gxui/math"
	"github.com/google/gxui/themes/dark"
)

func appMain(driver gxui.Driver) {
	theme := dark.CreateTheme(driver)

	font, err := driver.CreateFont(gxfont.Default, 75)
	if err != nil {
		panic(err)
	}

	window := theme.CreateWindow(380, 100, "Opacity")

	label := theme.CreateLabel()
	label.SetFont(font)
	label.SetText("Hello world")

	layout := theme.CreateLinearLayout()
	layout.SetBackgroundBrush(gxui.CreateBrush(gxui.Gray50))
	layout.AddChild(label)

	window.AddChild(layout)

	ticker := time.NewTicker(time.Millisecond * 30)
	phase := float32(0)
	go func() {
		for _ = range ticker.C {
			r := 0.75 + 0.25*math.Cosf((phase+0.000)*math.TwoPi)
			g := 0.75 + 0.25*math.Cosf((phase+0.333)*math.TwoPi)
			b := 0.75 + 0.25*math.Cosf((phase+0.666)*math.TwoPi)
			a := 0.5 + 0.5*math.Cosf(phase*10)
			phase += 0.01
			driver.Call(func() {
				label.SetColor(gxui.Color{R: r, G: g, B: b, A: a})
			})
		}
	}()

	window.OnClose(ticker.Stop)
	window.OnClose(driver.Terminate)
}

func main() {
	gl.StartDriver(appMain)
}
```

**Comment by ben-clayton** [comment]
[Added ```gxui/samples/hello_world```](https://github.com/google/gxui/commit/95ce27f0f2d6ce7491b23d3374ccd2db5991f23c).
